app-misc/anki/metadata.xml

```diff
@@ -13,4 +21,8 @@
 	<upstream>
 		<remote-id type="github">ankitects/anki</remote-id>
 	</upstream>
+	<use>
+		<flag name="lto">Build with link-time optimization. Ignored if debug is enabled.</flag>
+		<flag name="minimal">Only build the standalone sync server. Does not respect the lto flag.</flag>
```

Comment:
We really try to avoid USE=minimal if we can, as it's non-intuitive by name, and has confusing semantics. It's "negative USE logic".

Can't we do the positive version? USE=gui or whatever?